R/estimate_reporting.R

```diff
@@ -87,14 +119,16 @@ estimate_reporting <- function(df_in,
 
     df_severity <- df_severity[!is.na(df_severity$severity_me), ]
 
-    if (!is.null(max_date)) {
+    # collect the severity at the last date, or the date specified by
+    # the user in `max_date`
+    if (!missing(max_date)) {
```

Comment:
Again, `max_date` can't be missing because no default is provided. The best option might be to restore the default

Reply:
Fixed in https://github.com/epiverse-trace/datadelay/pull/44/commits/809c4557a1949bb4cc13f4516597076e0343f919 and others